src/routing/locations.ts

```diff
@@ -27,7 +28,12 @@ export const locations = {
   sceneDetail: (projectId = ':projectId') => `/scenes/${projectId}`,
   collections: () => '/collections',
   itemDetail: (itemId = ':itemId') => `/items/${itemId}`,
-  collectionDetail: (collectionId = ':collectionId') => `/collections/${collectionId}`,
+  collectionDetail: (collectionId = ':collectionId', type: CollectionType = CollectionType.DECENTRALAND) =>
+    type === CollectionType.DECENTRALAND
+      ? `/collections/${collectionId}`
+      : type === CollectionType.THIRD_PARTY
+      ? locations.thirdPartyCollectionDetail(collectionId)
+      : '',
```

Comment:
I'm thinking that it might be better to throw if the type doesn't match any of them to prevent any issues with the generation of urls.